content/en/tracing/profiling/getting_started.md

```diff
@@ -0,0 +1,229 @@
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.  |
+
+
+[1]: https://docs.oracle.com/javacomponents/jmc-5-4/jfr-runtime-guide/about.htm
+[2]: https://app.datadoghq.com/account/settings#agent/overview
+[3]: https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/java.html
+[4]: https://github.com/DataDog/dd-trace-java/releases/tag/v0.55.1
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python 3.5+. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1] or [6.20.2][1].
+
+2. Install `ddtrace` which contains both tracing and profiling:
+
+    ```shell
+    pip install ddtrace
+    ```
+
+     **Note**: Profiling is available in the `ddtrace` library for versions 0.36+.
+
+3. To automatically profile your code, import `ddtrace.profile.auto`. After import, the profiler starts:
+
+    ```python
+    import ddtrace.profiling.auto
+    ```
+
+4. After a minute or two, visualize your profiles on the [Datadog APM > Profiling page][2].
+
+**Note**:
+
+- Alternatively, profile your service by running it with the wrapper `pyddprofile`:
+
+    ```shell
+    $ pyddprofile server.py
+    ```
+
+- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use environment variables to set the parameters:
+
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_SERVICE`                                     | String        | The Datadog [service][3] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][4] name, for example `production`, which can be set here, or in `DD_PROFILING_TAGS` with `DD_PROFILING_TAGS="env:production"`. |
+| `DD_VERSION`                                     | String        | The version of your application.                             |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.   |
+
+<div class="alert alert-info">
+Recommended for advanced usage only.
+</div>
+
+- If you want to manually control the lifecycle of the profiler, use the `ddtrace.profiling.profiler.Profiler` object:
+
+    ```python
+    from ddtrace.profiling import Profiler
+
+    prof = Profiler()
+    prof.start()
+
+    # At shutdown
+    prof.stop()
+    ```
+
```

Comment:
Seems like the `fork` warning has disappeared. Not sure it's a good idea since customers complain that profiling does not work when they use e.g. a task manager like Celery.